src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -367,6 +367,8 @@ public bool Dispatch(EventArgs eventArgs, string nativeEventPayload)
 						args = _payloadConverter(nativeEventPayload);
 					}
 
+					args = args ?? new RoutedEventArgs { OriginalSource = this };
```

Comment:
Not sure to understand how does `args` may be `null` here

Reply:
I thought the changer where going to be in `WindowManager.ts`, in `getEventExtractor()`.
Here it should be an exception to have a null `args`.

Reply:
It would en up in being a null string, as there are no parameters for that event. What would you put there ?

Reply:
Should be better now :)